src/Form/Mapper/BlockFormMapper.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Form\Mapper;
+
+use Sonata\AdminBundle\Form\FormMapper as AdminFormMapper;
```

Comment:
We should not need any `AdminBundle` reference. We decoupled this bundle from the admin bundle on the 4.x branch

Reply:
This code is use only when user generate form for Block in Admin site.

Reply:
It would still need to require sonataAdmin on master https://github.com/sonata-project/SonataBlockBundle/blob/master/composer.json

The right way should be to add this code on sonataAdmin instead, if necessary.

Reply:
AFAIK we only need this mapping for the PageBundle:
https://github.com/sonata-project/SonataPageBundle/blob/3.x/src/Mapper/PageFormMapper.php

Reply:
So it could be directly added to the PageBundle then

Reply:
> So it could be directly added to the PageBundle then

Already done ;)